sixtyfps_runtime/corelib/Cargo.toml

```diff
@@ -42,7 +42,7 @@ weak-table =  "0.3"
 scopeguard = "1.1.0"
 cfg-if = "1"
 # Optional dependency to have conversion trait between femtovg::Color and sixtyfps-corelib::Color
-femtovg = { version = "0.1.3", optional = true }
+femtovg = { version = "0.2.0", optional = true, git = "https://github.com/tronical/femtovg", branch = "simon/filter" }
```

Comment:
ideally, we should remove this dependency.  The conversion can simply be done with an helper method.  (But this does not have to be done now)

Reply:
You're right, as "convenient" as it is, making updates here is becoming annoying :). I'll make this a separate cleanup later.

Reply:
It would be nice if there's a universal "color" crate that we can all use.